isisd/isis_nb_config.c

```diff
@@ -386,7 +386,7 @@ int isis_instance_advertise_high_metrics_modify(struct nb_cb_modify_args *args)
 
 	advertise_high_metrics = yang_dnode_get_bool(args->dnode, NULL);
 	area = nb_running_get_entry(args->dnode, NULL, true);
-	isis_area_advertise_high_metrics_set(area, advertise_high_metrics);
+	isis_area_advertise_high_metrics_set(area, advertise_high_metrics, NULL);
 
```

Comment:
The configuration does not take effect immediately.
It seems `lsp_regenerate_schedule(area, IS_LEVEL_1 | IS_LEVEL_2, 0);` is missing

Have we the same issue with the overload bit ?

`NULL` is for NULL pointer. `0 ` instead

Reply:
`lsp_regenerate_schedule()` is called within `isis_area_advertise_high_metrics_set()` -> `isis_circuit_metric_set()`

Reply:
> NULL is for NULL pointer. 0 instead
Which line(s) are you referring to? I used NULL in both places based on similar function calls

Reply:
Removed 3rd param from `isis_area_advertise_high_metrics_set()`